src/pet.js

```diff
@@ -70,16 +70,21 @@ function Pet() {
     },
 
     haveBaby(child){
-        child = new Pet ('Kitten'); //creates new pet with properties of Pet
-        this.children.push(child);
-        
-       /* function theChild(){
         Pet.call (this);
```

Comment:
What is `Pet.call(this) `doing?

Reply:
As per my research, it would let `child` inherit the properties of Pet.

Reply:
I actually wanted `child ` to inherit methods of Pet as well so I could manipulate it too, but I wasn't sure how to. So I tried creating another function within the haveBaby() and did some experimenting... which obviously didn't work lol. You can see some leftover evidence in the commented out part.